src/github.org

```diff
@@ -0,0 +1,104 @@
+#+TITLE: GitHub Training 2017
+#+AUTHOR: Sahit Chintalapudi and Josh Ting 
+#+EMAIL: schintalapudi@gatech.edu
+#+REVEAL_THEME: black
+#+REVEAL_TRANS: linear
+#+REVEAL_SPEED: fast
+#+REVEAL_PLUGINS: (notes pdf)
+#+REVEAL_HLEVEL: 1
+#+OPTIONS: toc:nil timestamp:nil reveal_control:t num:nil reveal_history:t tags:nil author:nil
+
+* What is Git?
+- Git is a "Distributed Version Control System" (DCVS)
+- It is the tool RoboJackets uses to collaborate on code
+- Git tracks changes made to files in your project, and handles merging
+  these changes
+- Working on different parts of code and then combining them creates unique challenges that
+  cannot be handled by file sharing systems like DropBox and Google Drive
+
+* What is GitHub?
+- GitHub is a publically hosted git server
+- It hosts our projects as "repositories" on the internet
+- This saves us the hassle of hosting a server for fellow RoboJackets to
+  obtain copies of the code
+- As you matriculate through Tech, you'll be building an online portfolio of
+  your work on GitHub.
+
+* How do you find things on GitHub?
+- Repositories are stored at https://github.com/<ORG-OR-USER-NAME>/<REPO-NAME>
+- So all of our projects can be found at https://github.com/RoboJackets
+- Your personal projects will exist at https://github.com/<YOUR-GH-USERNAME>
+
+** Create a GitHub account
+- [[https://github.com/join%0A][https://github.com/join]]
+- Please include /at least/ your real first name
+# Emphasize this requirement is for display name on GitHub, not username
+
+[[file:https://i.imgur.com/0cdXQXW.png]]
+
+* Fork and star our repo
+- Navigate to github.com/robojackets/robocup-software
+    - Or the repo of whatever RoboJackets team you are on
+
+[[file:https://i.imgur.com/kYzz2oh.png]]
+
+* What is a fork?
+- Problem: How do I try something out on a project without affecting the project itself?
+- Solution: Forks!
+- A fork is a copy of a repository that exists on your personal account
+- Changes made on a fork will not affect the original repository, and vice versa
+- Your fork is present on https://github.com/<YOUR-GH-USERNAME>/robocup-software
+
+* Install Git on your computer
```

Comment:
If you link to https://git-scm.com/downloads, it will display the appropriate download.

For linux, people should be using their package manager though, to avoid conflicts, and homebrew on mac.